src/auth/ktor/Principal.kt

```diff
@@ -16,12 +16,12 @@ import org.synthesis.keycloak.admin.KeycloakUserGroup
  */
 internal fun JWTCredential.principal(realm: KeycloakRealm, client: KeycloakApiClient): UserAccount = UserAccount(
     id = id(),
-    email = payload.getClaim("email").asString(),
+    email = "",
     groups = groups(),
```

Comment:
Is this left blank because we might not get the email from keycloak/the sign in partner anymore?

Reply:
Yes that is correct. Part of the information this expects is no longer in the token. Which is not an issue as it doesn't actually do anything with it. It only retrieves the id from this and then gathers all user information from the database (see SecurityConfiguration). I create a task on the backlog to change this implementation https://naturalis.atlassian.net/browse/DD-152